### PR TITLE
revert: feat(balance): changes in base zombie drops

### PR DESCRIPTION
**Pull request by RoyalFox2140**
### Required

- [x] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [x] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [x] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [x] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change

#5197 slashed firearm spawn rates by 70%, changed tool spawns by a large amount, and cut back on the amount of smokers and thus available morale boosts via cigs a lot. These changes were snuck in relatively quietly as the community wasn't informed at such a huge shift in loot, additionally After I spoke with Chaosvolt he seemed unaware of how significant the firearm spawns were slashed.

## Describe the solution

Revert. Something of this dramatic decision needs polling. I've talked to a number of people on the Discord, people notice there's a lot less guns but don't understand why, this was never communicated.

## Describe alternatives you've considered

Partial revert, but we'll work on it more as a group.

## Testing

It worked before this PR and it worked after. Tests will test.

## Additional context

### CODE REVIEW

File: data/json/itemgroups/Drugs_Tobacco_Alcohol/drugs.json [hunk 1]
```diff
@@ -181,11 +181,7 @@
       [ "cig", 60 ],
       [ "tobacco", 30 ],
       [ "chaw", 20 ],
-      [ "seed_tobacco", 5 ],
-      [ "cigar", 10 ],
-      [ "handrolled_cig", 10 ],
-      [ "ecig", 20 ],
-      [ "advanced_ecig", 10 ]
+      [ "seed_tobacco", 5 ]
```
Comment (chaosvolt) [inline data/json/itemgroups/Drugs_Tobacco_Alcohol/drugs.json:184]:
```suggestion
      [ "seed_tobacco", 5 ],
      [ "cigar", 10 ],
      [ "handrolled_cig", 10 ],
      [ "ecig", 20 ],
      [ "advanced_ecig", 10 ]
```

File: data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "id": "weapons",
     "items": [
       [ "chain", 20 ],
-      [ "knuckle_brass", 30 ],
+      [ "knuckle_brass", 20 ],
```
Comment (chaosvolt) [inline data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json:7]:
```suggestion
      [ "knuckle_brass", 30 ],
```

File: data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json [hunk 3]
```diff
@@ -23,16 +23,16 @@
       [ "throwing_axe", 6 ],
       { "item": "shuriken", "prob": 4, "count": [ 1, 4 ] },
       [ "punch_dagger", 6 ],
-      [ "pipe", 40 ],
-      [ "bat", 80 ],
+      [ "pipe", 20 ],
+      [ "bat", 60 ],
```
Comment (chaosvolt) [inline data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json:27]:
```suggestion
      [ "pipe", 40 ],
      [ "bat", 80 ],
```

File: data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json [hunk 4]
```diff
@@ -23,16 +23,16 @@
...
       [ "bullwhip", 10 ],
       [ "machete", 5 ],
       [ "baton", 8 ],
       [ "blackjack", 10 ],
       [ "tazer", 3 ],
       [ "rapier", 3 ],
       [ "cavalry_sabre", 2 ],
-      [ "bat_metal", 80 ],
+      [ "bat_metal", 60 ],
```
Comment (chaosvolt) [inline data/json/itemgroups/Weapons_Mods_Ammo/weapons_misc.json:35]:
```suggestion
      [ "bat_metal", 60 ],
```

**Review by chaosvolt — COMMENTED** [review]
Couple things in the revert seem okay to keep, the rest seems acceptable to revert.

**Comment by RoyalFox2140** [comment]
@chaosvolt Check it now, suggestions comitted

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**